lib/items/uploadSession.js

```diff
@@ -16,11 +17,14 @@ var userPathGenerator = require("../helpers/pathHelper");
  * @param {Number} params.fileSize Size of file
  * @param {Number} [params.chunksToUpload=20] Number of chunks to upload at a time
  * @param {Function} progress Get uploaded bytes callback
+ * @param {Boolean} writeStream Return writeStream instead of promise
+ * @param {Function} cb(err, results) Callback for writestream
  *
- * @return {Object} Item
+ * @return {Object | writeStream} Item
  */
 
-function uploadSession(params, progress = () => {}) {
+function uploadSession(params, progress, writeStream, cb) {
```

Comment:
`progress` is failing for me as it is required now. If you rebase from master it will default to `progress = () => {}` and wont break existing code